.github/workflows/code_size.yml

```diff
@@ -36,12 +35,10 @@ jobs:
     - name: Compute code size difference
       run: tools/metrics.py diff ~/size0 ~/size1 | tee diff
     - name: Save PR number
-      if: github.event_name == 'pull_request'
```

Comment:
Is it worth leaving these lines in, to make it clear what's going on, make it more robust, and just in case we want to reenable the push event?

Reply:
Sure, have reverted.